site/themes/s2b_hugo_theme/assets/js/cal/main.js

```diff
@@ -56,6 +56,34 @@ $(document).ready(function() {
                     exportURL.searchParams.set('id', value.id);
                     value.exportlink = exportURL.toString();
 
+                    /**
+                     * Add To Google Calendar
+                     */
+                    const googleCalUrl = new URL('https://www.google.com/calendar/render')
+                    const startDate = dayjs(`${value.date} ${value.time}`).toISOString()
+                    const duration = value.duration ?? 60 // Google requires a duration and defaults to 60 minutes anyway
+                    const endDate = dayjs(startDate).add(dayjs.duration({ 'minute': duration })).toISOString()
+                    /**
+                     * Matches anything that is not a word or whitespace
+                     * @example
+                     * "2025-05-21T16:30:00.000Z".replace(regex, '') // 20250521T163000000Z
+                    */
+                    const regex = /[^\w\s]/gi
+
+                    // Remove colons and periods for Google Calendar URL (2025-05-21T16:30:00.000Z => 20250521T163000000Z)
+                    const calendarDates = `${startDate.replace(regex, '')}/${endDate.replace(regex, '')}`
+
+                    googleCalUrl.search = new URLSearchParams({
+                      action: "TEMPLATE",
+                      text: `shift2Bikes: ${value.title}`,
+                      location: value.address,
+                      details: value.details,
```

Comment:
Could use `${value.details}\n\n${value.shareable}` to append a link to the event in the description